packages/fx-core/src/component/driver/teamsApp/validateTestCases.ts

```diff
@@ -99,7 +99,39 @@ export class ValidateWithTestCasesDriver implements StepDriver {
         return err(appStudioTokenRes.error);
       }
       const appStudioToken = appStudioTokenRes.value;
-
+      // Check if the app has ongoing validation
+      const existingValidationResponse = await AppStudioClient.getAppValidationRequestList(
+        manifest.id,
+        appStudioToken
+      );
+      if (existingValidationResponse.appValidations) {
+        for (const validation of existingValidationResponse.appValidations) {
+          if (validation.status === AsyncAppValidationStatus.InProgress) {
```

Comment:
Other status may be considered, e.g. Created

Reply:
Yes, I will add it. Thanks